substrate/client/offchain/Cargo.toml

```
@@ -22,15 +22,15 @@ codec = { features = ["derive"], workspace = true, default-features = true }
 fnv = { workspace = true }
 futures = { workspace = true }
 futures-timer = { workspace = true }
-hyperv14 = { features = [
-	"http2",
-	"stream",
-], workspace = true, default-features = true }
+http-body-util = { workspace = true }
+hyper = { features = ["http2"], workspace = true, default-features = true }
 hyper-rustls = { features = ["http2"], workspace = true }
```

we already enable http1 and http2 globally in the workspace

```suggestion
hyper-rustls = { workspace = true }
```